Client/WarFare/UIImageTooltipDlg.cpp

```diff
@@ -1052,6 +1053,79 @@ int	CUIImageTooltipDlg::CalcTooltipStringNumAndWrite(__IconItemSkill* spItem, bo
 	}
 	ERROR_EXCEPTION
 
+	if (spItem->pItemBasic != nullptr)
+	{
+		std::string szRemark = spItem->pItemBasic->szRemark;
+	
+		int strSize = spItem->pItemBasic->szRemark.size();
+		if (strSize > 0)
+		{
+			int splitPos = 0;
+			for (int i = 0; i < strSize; i++)
+			{
+				if (szRemark[i] == '(' || (szRemark[i] == ' ' && i > strSize / 2))
+				{
+					splitPos = i;
+					break;
+				}
+			}
+			if (splitPos > 0)
+			{
+				m_pStr[iIndex]->SetColor(m_CWhite);
+				m_pstdstr[iIndex] = fmt::format("*{}", szRemark.substr(0, splitPos));
+				m_pStr[iIndex]->SetStyle(UI_STR_TYPE_HALIGN, UISTYLE_STRING_ALIGNCENTER);
+
+				iIndex++;
+
+				m_pStr[iIndex]->SetColor(m_CWhite);
```

Comment:
iIndex is only "safe" for the first one. ERROR_EXCEPTION catches it (prior to its usage) and skips to `exceptions` if it'll be out of range. The second one *isn't* checked/safe.

As loathe as I am to say it, it will need the ERROR_EXCEPTION macro here to ensure it behaves.